lib/awful/widget/tasklist.lua

```diff
@@ -83,10 +104,17 @@ local instances
 -- @beautiful beautiful.tasklist_bg_image_minimize
 -- @tparam[opt=nil] string bg_image_minimize
 
--- Disable the tasklist clients icons.
+--- Disable the tasklist clients icons.
 -- @beautiful beautiful.tasklist_tasklist_disable_icon
 -- @tparam[opt=false] boolean tasklist_disable_icon
 
+--- Disable the extra tasklist client property notification icons.
+--
+-- See <a href="#client_icons">the top of this page</a> for more details.
```

Comment:
Sorry, I've meant instead of "the top of this page", so the link text, not `href`.  (but you could use `%20` there to encode spaces)
But feel free to ignore that: it is not as bad as the typical "here", which is unspecific.